### ISSUE TITLE
[A11y][TFM]TFM selection filters do not announce expanded/collapsed state

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9451]
Url: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1751374

Environment Details:
Application Name: Filtering by Target Frameworks
#URL: [NuGet Gallery | Packages (nugettest.org)](https://dev.nugettest.org/packages?q=)
Windows Version: Win11
OS Version: 22621.1105
Chromium Edge Version: Version 109.0.1518.78 (Official build) (64-bit)
Chrome Version: Version 110.0.5481.77 (Official Build) (64-bit)
Screen reader version: Narrator/ NVDA 2022.2

Repro Steps: 
Hit the URL: [NuGet Gallery | Packages (nugettest.org)](https://dev.nugettest.org/packages?q=)
Tab to Frameworks section.
Tab to the collapse/expand buttons present in the Frameworks section.
Turn on Narrator/NVDA.
Observe that screen reader is not announcing the state whether collapsed/expanded or hidden/shown for the control 'shows and hides TFM filter'.

User Impact:
Screen reader users will be confused if the state of the control is not announced.

Notes:
This would appear to be due to the keyboard focused element not being properly tagged with aria-expanded. Adding this to the proper control element should resolve this.

**Comment by theot12** [comment]
AzDo issue closed.
